core/model/messages/SynchronizeParametersMessage.h

```diff
@@ -14,27 +14,25 @@
  * limitations under the License.
  */
 
-#ifndef SUBDEVICEREGISTRATIONRESPONSE_H
-#define SUBDEVICEREGISTRATIONRESPONSE_H
+#ifndef WOLKABOUTCORE_SYNCHRONIZEPARAMETERSMESSAGE_H
+#define WOLKABOUTCORE_SYNCHRONIZEPARAMETERSMESSAGE_H
 
-#include "core/model/PlatformResult.h"
-
-#include <string>
+#include "core/model/messages/MessageObject.h"
 
 namespace wolkabout
 {
-class SubdeviceRegistrationResponse
+class SynchronizeParametersMessage : public MessageObject
```

Comment:
Does this message exist in protocol?

Reply:
Yes, it got added on topic `d2p/{deviceKey}/synchronize_parameters`.